src/rdf_graph/build_graph.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+
+from datetime import datetime
+
+from rdf_graph import RDFGraph
```

Comment:
Hey Ben, do we want this code to be able to run from the base of the repo? Or are we running from the 'rdf_graph' folder?

Reply:
Good catch - making that import absolute and not relative should be more robust.